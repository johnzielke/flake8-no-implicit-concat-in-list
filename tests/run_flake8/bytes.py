b = [b"aaa",
     b"bbb"
     b"ccc"]
