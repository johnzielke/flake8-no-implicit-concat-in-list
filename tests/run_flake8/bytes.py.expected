./bytes.py:2:12: ICL122 Implicitly concatenated bytes literals in list literal over multiple lines.
